include/measurement_kit/common/utils.hpp

```diff
@@ -29,6 +29,7 @@ T split(std::string s, std::string pattern = "\\s+") {
     return {first, last};
 }
 
+int strcasecmp_wrapper(const char *, const char *);
```

Comment:
Need this wrapper because previously the public http header was pulling directly `strcasecmp` which is a function that is not available on Windows. Better to pull the wrapper and use it to redirect to impl.